addons/account/static/src/components/tax_totals/tax_totals.js

```diff
@@ -143,7 +143,7 @@ export class TaxTotalsComponent extends Component {
     }
```

Comment:
Could you rewrite your message. It would be nice to have a description of the problem. Then an explanation of why it is linked to the fact that two references are the same.